packages/image/README.md

```diff
@@ -0,0 +1,126 @@
+
+> Note that the `src` must start with a `.`. `@sveltejs/image` includes a preprocessor that will convert it to the following:
+>
+> ```svelte
+> <script>
+> 	import Image from '@sveltejs/image';
+> 	import YourImage from './path/to/your/image.jpg';
+> </script>
+>
+> <Image src={YourImage} alt="An alt text" />
+> ```
+
+### Choosing static vs dynamic
+
+Using the static provider generates the images at build time, so that build time may take longer the more images you transform.
+
+Using an image CDN provides more flexibility with regards to sizes and you can pass image sources not known at build time, but it comes with potentially a bit setup overhead (configuring the image CDN) and possibly usage cost.
+
+You can mix and match both solutions in one project.
+
+### `Image` component props
+
+There are a few things you can customize:
+
+- `priority`: set this to `true` for the most important/largest image on the page so it loads faster
+- `sizes`: If your image is less than full width on one or more screen sizes, add this info here. When using dynamic providers the widths can be adjusted accordingly to produce a more optimal `srcset`. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes) for more info on the attribute
+- `style`: to style the image
+- `class`: to style the image. Be aware that you need to pass classes that are global (i.e. wrapped in `:global()` when coming from a `<style>` tag)
+- `loading`: loading behavior of the image. Defaults to `lazy` which means it's loaded only when about to enter the viewport. Set to `eager` to load right away (the default when setting `priority`)
+- `provider`: you can pass more than the `default` provider in `vite.config.js`. If you then want to use a different provider than the `default` one, pass it here
+- `providerOptions`: provider-specific image CDN options. For example `quality` for Vercel
+
+## Best practices
+
+- Always provide a good `alt` text
+- Your original images should have a good quality/resolution. Images are easier to size down than up
+- Choose one image per page which is the most important/largest one and give it `priority` so it loads faster. This gives you better web vitals scores (largest contentful paint in particular)
+- Give the image a container or a styling so that it is constrained and does not jump around. `width` and `height` help the browser reserving space while the image is still loading
+
+## Roadmap
+
+This is an experimental MVP for getting initial feedback on the implementation/usability of an image component usable with SvelteKit (can also be used with Vite only). Once the API is stable, we'll want to create a more seamless integration with SvelteKit, i.e. less setup required.
+
+## Acknowledgements
+
+We'd like to thank the authors of the Next/Nuxt/Astro/`unpic` image components for inspiring this work.
```

Comment:
should we mention `svelte-preprocess-import-assets` and `vite-imagetools` or omit them since we're a bit involved with them?

Reply:
We definetly should also mention them!